RELEASE_PROCESS.md

```diff
@@ -27,43 +27,121 @@ In order to release, you will need the following accounts/permissions:
+- Enter the username and password to test the login.  The username and password are stored in the following script:  
+https://github.com/chef/chef-server/blob/master/terraform/common/files/add_user.sh.  
+- Verify that the login is successful.
+- Navigate to `http://<hostname>/id` via web browser where _hostname_ is the DNS name of the emphemeral machine obtained in the previous step.
+- Login with the same user/password as the previous login step above.
+- Select the 'signed in as <username>' link in the upper-left. Confirm that the email is grayed-out.
+- Clean-up:
+```
+PLATFORM=ubuntu-18.04 INSTALL_VERSION=<version> UPGRADE_VERSION=<version> SCENARIO=standalone-fresh-install ENABLE_ADDON_PUSH_JOBS=false ENABLE_GATHER_LOGS_TEST=false ENABLE_PEDANT_TEST=false ENABLE_PSQL_TEST=false ENABLE_SMOKE_TEST=false ENABLE_IPV6=true make destroy
+```
+Typical scenario for Azure, where _version_ is the version number of the release candidate you are testing:
+```
+$ cd umbrella/chef-server/scenarios/azure
+$ ARM_DEPT=Eng ARM_CONTACT=your_login_here make create-resource-group
+...
+$ PLATFORM=ubuntu-18.04 INSTALL_VERSION=<version> UPGRADE_VERSION=<version> SCENARIO=external-postgresql ENABLE_ADDON_PUSH_JOBS=false ENABLE_GATHER_LOGS_TEST=false ENABLE_PEDANT_TEST=false ENABLE_PSQL_TEST=false ENABLE_SMOKE_TEST=false ENABLE_IPV6=true make apply
+```
+- Perform the same login processes specified above for AWS.
+- Clean-up:
+```
+PLATFORM=ubuntu-18.04 INSTALL_VERSION=<version> UPGRADE_VERSION=<version> SCENARIO=external-postgresql ENABLE_ADDON_PUSH_JOBS=false ENABLE_GATHER_LOGS_TEST=false ENABLE_PEDANT_TEST=false ENABLE_PSQL_TEST=false ENABLE_SMOKE_TEST=false ENABLE_IPV6=true make destroy
+```
+
+Any failures must be fixed before shipping a release, unless they are "known failures" or expected. A document at an unknown state of updatedness tracking known failures can be found at:  
+https://docs.google.com/spreadsheets/d/10LZszYuAIlrk1acy0GRhrZMd0YohLTQWmcNIdY6XuZU/edit#gid=0  
+Use chef.io account credentials to access it.
+Note that no changes other than CHANGELOG/RELEASE_NOTES changes should land on master between testing and releasing since we typically tag HEAD of master. If something large does land on master, the release tag you create should point specifically at the build that you tested. The git SHA of the build you are testing can be found in /opt/opscode/version-manifest.json.
+
+### Informing everyone of a pending release pre-promote
+
+- [ ] Announce your intention to release to the following slack channels  
+    - #a2-release-coordinate
+    - #chef-server  
+Sample pre-promote announcement:  
+We are planning to do a release of Chef Infra Server X.Y.Z shortly. Details can be found at: https://github.com/chef/chef-server/wiki/Pending-Release-Notes
+
+### Preparing for the Release
 
 - [ ] Check RELEASE_NOTES.md to ensure that it describes the
```

Comment:
This step is covered above and should be removed. We manage these in the wiki now

Reply:
No announcement?  Roger that.

Reply:
Assuming I delete the entire announce section.